sycl/include/sycl/handler.hpp

```diff
@@ -2541,6 +2541,7 @@ class __SYCL_EXPORT handler {
   std::shared_ptr<detail::handler_impl> MImpl;
   std::shared_ptr<detail::queue_impl> MQueue;
   std::shared_ptr<ext::oneapi::experimental::detail::graph_impl> MGraph;
+  std::shared_ptr<ext::oneapi::experimental::detail::node_impl> MSubgraphNode;
```

Comment:
What if we do multiple layers of nesting: graph -> graph -> graph. My understanding is that this is supported because the handler always sees one level step by step, correct?